BO4E/BO/Auftrag.cs

```diff
@@ -0,0 +1,100 @@
+    public abstract class Auftrag : BusinessObject
+    {
+        /// <summary>
+        /// workaround
+        /// </summary>
+        [System.Text.Json.Serialization.JsonIgnore]
+        [Newtonsoft.Json.JsonIgnore]
+        [ProtoMember(4, Name = nameof(Ausfuehrungsdatum))]
+        [CompatibilityLevel(CompatibilityLevel.Level240)]
+        protected DateTime _Ausfuehrungsdatum
+        {
+            get => Ausfuehrungsdatum.UtcDateTime;
+            set => Ausfuehrungsdatum = DateTime.SpecifyKind(value, DateTimeKind.Utc);
+        }
+
+        /// <summary>
+        /// Das Ausführungsdatum beschreibt zu welchem Zeitpunkt ein Auftrag ausgeführt werden soll.
+        /// </summary>
+        [JsonProperty("ausfuehrungsdatum", Required = Required.Always)]
+        [JsonPropertyName("ausfuehrungsdatum")]
+        [ProtoIgnore]
+        public DateTimeOffset Ausfuehrungsdatum { get; set; }
+
+        /// <summary>
+        /// workaround
+        /// </summary>
+        [System.Text.Json.Serialization.JsonIgnore]
+        [Newtonsoft.Json.JsonIgnore]
+        [ProtoMember(5, Name = nameof(Fertigstellungsdatum))]
+        [CompatibilityLevel(CompatibilityLevel.Level240)]
+        protected DateTime _Fertigstellungsdatum
+        {
+            get => Fertigstellungsdatum.UtcDateTime;
+            set => Fertigstellungsdatum = DateTime.SpecifyKind(value, DateTimeKind.Utc);
+        }
+
+        /// <summary>
+        /// Das Fertigstellungsdatum beschreibt zu welchem Zeitpunkt ein Auftrag ausgeführt wurde/wird.
+        /// </summary>
+        [JsonProperty("fertigstellungsdatum", Required = Required.Default)]
+        [JsonPropertyName("fertigstellungsdatum")]
+        [ProtoIgnore]
+        public DateTimeOffset Fertigstellungsdatum { get; set; }
+
+        /// <summary>
+        /// Die Adresse, die sich in Belieferung befindet.
+        /// </summary>
+        [JsonProperty("lieferanschrift", Required = Required.Default)]
+        [JsonPropertyName("lieferanschrift")]
+        [ProtoMember(6)]
+        public Adresse Lieferanschrift { get; set; }
+
+        /// <summary>
+        /// Die ID der Marktlokation der der zu sperrende Zähler zugeordnet ist.
+        /// </summary>
+        [JsonProperty("marktlokationsId", Required = Required.Always)]
+        [JsonPropertyName("marktlokationsId")]
+        [ProtoMember(7)]
+        public string MarktlokationsId { get; set; }
+
+        /// <summary>
+        /// Ein zusätzlicher Freitext
+        /// </summary>
+        [JsonProperty("bemerkung", Required = Required.Default)]
+        [JsonPropertyName("bemerkung")]
+        [ProtoMember(8)]
+        public string Bemerkung { get; set; }
+
+        /// <summary>
+        /// Die Mindestkosten eines Auftrags (z.B. für eine Sperrung)
+        /// </summary>
+        [JsonProperty("minKosten", Required = Required.Default)]
+        [JsonPropertyName("minKosten")]
+        [ProtoMember(9)]
+        public Kosten MinKosten { get; set; }
```

Comment:
Hier können wir auch die paar Zeichen extra spendieren und MindestKosten ausschreiben finde ich. Sollten die Kosten aber nicht vielleicht in ein eigenes COM was optional am Auftrag hängt?

Reply:
> Sollten die Kosten aber nicht vielleicht in ein eigenes COM was optional am Auftrag hängt?

- Ein eigenes COM ist es ja schon (oder meinst du ein anderes als das `COM.Kosten`?)
- es hängt schon optional am Auftrag

Reply:
https://github.com/Hochfrequenz/BO4E-dotnet/pull/90/commits/200f9962715059dc40aebebbdb4c80c6adc5e857